marketplaceItems/vendingMachine/README.md

```diff
@@ -0,0 +1,29 @@
+# Vending Machine
+
+## Description
+This is the code repo for the Vending Machine Blueprint from High Fidelity.  This item allows you as a content creator to provide quick access to marketplace items you wish to promote, whether it's your own or someone else's!  By clicking or triggering on any of the buttons, a user will bring up the marketplace checkout page for that item and have it delivered to their inventory. The vending machine does not rez items, so user permissions for your domain will not be affected!
+
```

Comment:
The wording seems off....maybe "The vending machine does not rez items, so users will not require those permissions!"